service/user.go

```diff
@@ -147,6 +147,16 @@ func (uis *UIServer) userGetKey(w http.ResponseWriter, r *http.Request) {
 
 func (uis *UIServer) logout(w http.ResponseWriter, r *http.Request) {
 	uis.umconf.ClearCookie(w)
+	if uis.Settings.Ui.ExpireLoginCookieDomain != "" {
```

Comment:
Does this set it, not clear it? This is the same code as in login.

Reply:
its sets the expiration to -1hr, forcing the browser to delete the cookie (we can't directly delete a cookie from the browser)